js/adapt-contrib-instructionError.js

```diff
@@ -29,6 +40,44 @@ class InstructionError extends Backbone.Controller {
     notify.popup(notifyObject);
   }
 
+  showInlineError(model) {
+    const data = model.toJSON();
+    this.addErrorClass(model);
+
+    // Update instruction text
+    const dataWithInitial = Object.assign(data, { instruction: model.get('instructionInitial') });
+    const instruction = Handlebars.compile(this.config.body)(dataWithInitial);
+    model.set('instruction', instruction);
+
+    // Focus on instruction text element
+    const $instruction = $(`.${data._id}`).find('.component__instruction').first();
+    a11y.focusFirst($instruction, { defer: true });
+  }
+
+  addErrorClass(model) {
+    const data = model.toJSON();
+    const classes = [
+      data._classes,
+      'has-error'
+    ].filter(Boolean).join(' ');
+    model.set('_classes', classes);
+  }
```

Comment:
This will add duplicate `has-error` classes because it doesn't ensure that classes are unique.

Reply:
Love it. Super useful.